eq-author-api/schema/tests/subscription.test.js

```diff
@@ -120,8 +138,8 @@ describe("subscriptions", () => {
         id: pageId,
         title: "",
       });
-      const p = iterator.next();
-      const result = await Promise.race([p, wait(50)]);
+      const iteratorNext = iterator.next();
```

Comment:
p is fine here, it's pretty accepted shorthand for a promise, if we want to be more descriptive a name with the word promise in it would work.

Reply:
I will change it to `promise` 👍